statesync/messages.go

```diff
@@ -16,6 +17,9 @@ const (
 	chunkMsgSize = int(16e6)
 )
 
+// assert Wrapper interface implementation of the state sync proto message type.
+var _ p2p.Wrapper = (*ssproto.Message)(nil)
+
 // mustEncodeMsg encodes a Protobuf message, panicing on error.
 func mustEncodeMsg(pb proto.Message) []byte {
```

Comment:
you should be able to remove this, the test can use the wrap function you defined in `/proto/tendermint/statesync`